src/modules/userinvolve.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+import foodsUI from "./foodsUI.js";
+
+
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const key = '4ZfW9Nw9KyQJZ8SztYPH';
```

Comment:
[OPTIONAL] Following the naming convention in JavaScript best practices, the constants should be written in uppercase letters. (ex: URL, KEY ...).